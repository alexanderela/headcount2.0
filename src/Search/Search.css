.Search {
	width: 100%;
	display: flex;
	justify-content: center;
	align-items: center;
}

.search-input {
	width: 25rem;
  border-radius: 10px;
  height: 2rem;
  margin: 0 0 2rem 0;
}