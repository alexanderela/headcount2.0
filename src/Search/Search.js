import React, { Component } from 'react';
import './Search.css';
import PropTypes from 'prop-types'

class Search extends Component {
	constructor(props) {
		super(props);
		this.state = {

		}
	}

	render() {
		return(
			<div>
			</div>
		)
	}
}



export default Search;