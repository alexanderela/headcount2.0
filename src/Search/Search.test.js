import React from 'react';
import ReactDOM from 'react-dom';
import Search from './Search.js';
import { shallow } from 'enzyme';

describe('Search', () => {
	let wrapper

	beforeEach(() => {
		wrapper = shallow(<Search />)
	})

	it('matches the snapshot', () => {
		expect(wrapper).toMatchSnapshot()
	})

	it('renders only the district searched by user')
		// expect wrapper id to be x
		// expect wrapper length to be 1
})