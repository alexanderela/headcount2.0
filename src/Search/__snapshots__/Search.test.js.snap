// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search matches the snapshot 1`] = `undefined`;
